backend/app/v1/auth/auth_router.py

```diff
@@ -52,7 +52,7 @@ async def login(user: UserLogin, request: Request, response: Response) -> Sessio
 
 	except InvalidCredentialsError as _:
 		logger.info(f"Invalid credentials for {user.email}")
-		raise HTTPException(status_code=404, detail="Invalid credentials") from None
+		raise HTTPException(status_code=401, detail="Invalid credentials") from None
```

Comment:
good job, sad thing we have to clean this mess up after bakusie sometimes cuz they cannot do their job.
@Memetelve @malekk1 
I am sending you back to great site like Mr. new man said.
https://http.cat/